testing/trino-server-dev/etc/catalog/delta.properties

```diff
@@ -0,0 +1,22 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used
+# in production. For example configuration, see the Trino documentation.
+#
+
+connector.name=delta-lake
+
+hive.metastore.uri=thrift://localhost:9083
```

Comment:
Add a comment how to start appropriate environment.
like https://github.com/trinodb/trino/blob/022a266cf6ac8eea18ac4319a0591c2e41d9352c/testing/trino-server-dev/etc/catalog/hive.properties#L10-L12

Reply:
I haven't found any product test environment that comes bundled with MinIO

Reply:
will https://github.com/trinodb/trino/pull/11565 add this?

Reply:
By spinning up : 

```
testing/bin/ptl env up --environment EnvSinglenodeDeltaLakeOss --without-trino
```

and adding the following content to `delta.properties`

```
# minio uses port 9000 by default, which conflicts with hadoop
hive.s3.endpoint=http://localhost:9080
```


the configuration will work for `delta` connector.

Reply:
applied changes now that #11565 is merged.